web/src/beta/features/ProjectSettings/innerPages/PluginSettings/hooks.ts

```diff
@@ -0,0 +1,186 @@
+import { useApolloClient } from "@apollo/client";
+import { useCallback, useMemo } from "react";
+
+// TODO: migrate to services API
+import {
+  useGetInstalledPluginsQuery,
+  useInstallPluginMutation,
+  useUninstallPluginMutation,
+  useUploadPluginMutation,
+  useUpgradePluginMutation,
+} from "@reearth/classic/gql/graphql-client-api";
+import { useT, useLang } from "@reearth/services/i18n";
+import { useNotification } from "@reearth/services/state";
+
+export type Plugin = {
+  fullId: string;
+  id: string;
+  version: string;
+  title?: string;
+  author?: string;
+};
+
+export default ({ projectId, sceneId }: { projectId: string; sceneId?: string }) => {
+  const [installPluginMutation] = useInstallPluginMutation();
+  const [uploadPluginMutation] = useUploadPluginMutation();
+  const [uninstallPluginMutation] = useUninstallPluginMutation();
+  const [upgradePluginMutation] = useUpgradePluginMutation();
```

Comment:
Don't use graphql directly. Setup the mutation use through services/api 🙏🏻

Reply:
Updated.